scylla/metadata.csv

```diff
@@ -16,25 +17,61 @@ scylla.cache.partition_misses,count,,miss,,number of partitions needed by reads
 scylla.cache.partitions,gauge,,,,total number of cached partitions,0,scylla,cache partitions,
 scylla.cache.pinned_dirty_memory_overload,count,,byte,,amount of pinned bytes that we tried to unpin over the limit. This should sit constantly at 0; and any number different than 0 is indicative of a bug,-1,scylla,cache pinned dirty memory overload,
+scylla.cache.range_tombstone_reads,count,,,,total amount of range tombstones processed during read,0,scylla,,
 scylla.cache.reads,count,,get,,number of started reads,1,scylla,cache reads,
 scylla.cache.reads_with_misses,count,,read,,number of reads which had to read from sstables,0,scylla,cache reads with misses,
 scylla.cache.row_evictions,count,,eviction,,total number of rows evicted from cache,0,scylla,cache row evictions,
 scylla.cache.row_hits,count,,hit,,total number of rows needed by reads and found in cache,0,scylla,cache row hits,
 scylla.cache.row_insertions,count,,set,,total number of rows added to cache,0,scylla,cache row insertions,
 scylla.cache.row_misses,count,,miss,,total number of rows needed by reads and missing in cache,0,scylla,cache row misses,
 scylla.cache.row_removals,count,,eviction,,total number of invalidated rows,0,scylla,cache row removals,
+scylla.cache.row_tombstone_reads,count,,,,total amount of row tombstones processed during read,0,scylla,,
 scylla.cache.rows,gauge,,row,,total number of cached rows,0,scylla,cache rows,
+scylla.cache.rows_compacted_with_tombstones,count,,,,Number of rows scanned during write of a tombstone for the purpose of compaction in cache,0,scylla,,
+scylla.cache.rows_dropped_by_tombstones,count,,,,Number of rows dropped in cache by a tombstone write,0,scylla,,
 scylla.cache.rows_dropped_from_memtable,count,,row,,total number of rows in memtables which were dropped during cache update on memtable flush,0,scylla,cache rows dropped from memtable,
 scylla.cache.rows_merged_from_memtable,count,,row,,total number of rows in memtables which were merged with existing rows during cache update on memtable flush,0,scylla,cache rows merged from memtable,
 scylla.cache.rows_processed_from_memtable,count,,row,,total number of rows in memtables which were processed during cache update on memtable flush,0,scylla,cache rows processed from memtable,
 scylla.cache.sstable_partition_skips,count,,,,number of times sstable reader was fast forwarded across partitions,0,scylla,cache sstable partition skips,
 scylla.cache.sstable_reader_recreations,count,,,,number of times sstable reader was recreated due to memtable flush,0,scylla,cache sstable reader recreations,
 scylla.cache.sstable_row_skips,count,,row,,number of times sstable reader was fast forwarded within a partition,0,scylla,cache sstable row skips,
 scylla.cache.static_row_insertions,count,,row,,total number of static rows added to cache,0,scylla,cache static row insertions,
+scylla.cdc.operations_failed,count,,,,number of failed CDC operations,0,scylla,,
```

Comment:
Hmm I think by default all counter metrics come in with the affix `.count` have you tested these out?